components/tabs/Tabs.js

```diff
@@ -147,10 +180,18 @@ const factory = (Tab, TabContent) => {
       );
       return (
         <div ref='tabs' data-react-toolbox='tabs' className={classes}>
-          <nav className={theme.navigation} ref='navigation'>
-            {this.renderHeaders(headers)}
-          </nav>
-          <span className={theme.pointer} style={this.state.pointer} />
+          <div className={theme.navigationContainer}>
+            <div className={classnames(theme.arrowContainer, { [theme.invisible]: !(this.state.arrows.left || this.state.arrows.right) })} onClick={this.scrollNavigationLeft}>
+              <span className={classnames('material-icons', theme.arrow, { [theme.invisible]: !this.state.arrows.left })}>keyboard_arrow_left</span>
+            </div>
+            <nav className={theme.navigation} ref='navigation'>
+              {this.renderHeaders(headers)}
+              <span className={theme.pointer} style={this.state.pointer} />
+            </nav>
+            <div className={classnames(theme.arrowContainer, { [theme.invisible]: !(this.state.arrows.left || this.state.arrows.right) })} onClick={this.scrollNavigationRight}>
+              <span className={classnames('material-icons', theme.arrow, { [theme.invisible]: !this.state.arrows.right })}>keyboard_arrow_right</span>
```

Comment:
why doese it shows shows `keyboard_arrow_right`?

Reply:
It's a material designs icon: https://material.io/icons/#ic_keyboard_arrow_right

Reply:
Ah! Thank you. I saw the raw string sometimes when resizing `<Tabs />`